test/interp/simd-shift.txt

```diff
@@ -10,8 +10,44 @@
     v128.const i32 0xff000001 0xe0000002 0x00000003 0x00000004
     i32.const 11
     i8x16.shl)
+
+  ;; i16x8 shl
+  (func (export "i16x8_shl_0") (result v128)
+    v128.const i32 0xff000071 0xe0000702 0x00000003 0x00000004
+    i32.const 3
+    i16x8.shl)
+  (func (export "i16x8_shl_1") (result v128)
+    v128.const i32 0xff000071 0xe0000702 0x00000003 0x00000004
+    i32.const 19
+    i16x8.shl)
+
+  ;; i32x4 shl
+  (func (export "i32x4_shl_0") (result v128)
+    v128.const i32 0xff000071 0xe0000702 0x00000003 0x00000004
```

Comment:
please modify this test to show bits shifting in the full 32-bit range -- this output is identical to the i16x8 shift so it is hard to see that it is working.

Reply:
Done.